dotnet-setversion/Program.cs

```diff
@@ -9,24 +9,54 @@ class Program
     {
         static int Main(string[] args)
         {
-            if (args.Length < 1)
+            string csprojFile;
+            string versionString;
+
+            bool IsHelpFlag(string arg) => arg == "-h" || arg == "--help";
+            bool IsVstsFlag(string arg) => arg == "--vsts";
+            void PrintUsage() => Console.WriteLine("Usage: (dotnet setversion <version>) || (dotnet setversion <csproj-path> <version> --vsts)");
```

Comment:
In my opinion it's more common to have options (like `--vsts`) before arguments and optional arguments (like `csproj-path`) after required ones. I propose the following usage: `dotnet setversion [--vsts] <version> [<csproj-path>]`

Reply:
Yeah that seems reasonable.